javascript/ql/lib/semmle/javascript/frameworks/WebSocket.qll

```diff
@@ -81,10 +95,10 @@ module ClientWebSocket {
   /**
    * A client WebSocket instance.
    */
-  class ClientSocket extends EventEmitter::Range, DataFlow::NewNode, ClientRequest::Range {
+  class ClientSocket extends EventEmitter::Range, API::NewNode, ClientRequest::Range {
```

Comment:
FYI this class _won't_ have to be deprecated, because `API::NewNode` is a subtype of `DataFlow::NewNode`, and likewise for `API::CallNode`.